src/nvim/buffer.c

```diff
@@ -77,8 +77,6 @@
 #include "nvim/os/time.h"
 #include "nvim/os/input.h"
 
-#define HAVE_BUFLIST_MATCH
```

Comment:
I removed the `HAVE_BUFLIST_MATCH` define since it was apparently useless, please double check.